### ISSUE TITLE
Allow audio/mpeg streams next to m3u8 playlists

### ISSUE THREAD

**Issue by raboof** [fennifith/Alarmio#86]
For example the streams of the Dutch public radio, such as http://icecast.omroep.nl/radio2-bb-mp3, directly produce an `audio/mpeg` stream instead of wrapping them in an m3u8 playlist first. It would be cool if that was supported as well.

**Comment by raboof** [comment]
I did some digging: Alarmio uses the [HLS media source](https://exoplayer.dev/media-sources.html). When I switched to https and used the ProgressiveMediaSource instead (https://pastebin.com/VBYavgPu) I could indeed hear the stream above.

It would be nice to support both - should we just catch the UnrecognizedInputFormatException from the HLS source and then try the Progressive one instead? Not very sophisticated, but might be OK?
